src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1709,11 +1712,33 @@ private void HandleException(_EXCEPTION_POINTERS* pExceptionPointers)
             throw new NotSupportedException("HandleException");
         }
 
+        private bool runWithErrorTrap(void* function, void* parameter)
+        {
+            // This method is completely handled by the C++ wrapper to the JIT-EE interface,
+            // and should never reach the managed implementation.
+            Debug.Assert(false, "CorInfoImpl.runWithErrorTrap should not be called");
+            throw new NotSupportedException("runWithErrorTrap");
+        }
+
         private void ThrowExceptionForJitResult(HRESULT result)
         { throw new NotImplementedException("ThrowExceptionForJitResult"); }
         private void ThrowExceptionForHelper(ref CORINFO_HELPER_DESC throwHelper)
         { throw new NotImplementedException("ThrowExceptionForHelper"); }
 
+        private uint SizeOfPInvokeTransitionFrame
+        {
+            get
+            {
+                // struct PInvokeTransitionFrame:
+                //  m_RIP
+                //  m_FramePointer
+                //  m_pThread
+                //  m_dwFlags + align
+                //  m_PreserverRegs - RSP
```

Comment:
It looks like you only ever put RSP in the frame and then inhibit the JIT from enregistering a GC ref across a PInvoke -- it would be useful to clearly state that in here.